internal/common/machineid_test.go

```diff
@@ -0,0 +1,59 @@
+package common
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetMachineID(t *testing.T) {
+	t.Parallel()
+
+	f, err := os.CreateTemp(t.TempDir(), "")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
you can change the instances of `t.Fatal` by simply using `require.<checker>`. e.g. in this case you could use `require.NoError(t, err, ...)`

Reply:
Done.